R/app_server.R

```diff
@@ -7,6 +7,7 @@
 app_server <- function(input, output, session) {
   # Your application server logic
   moduleServer("00_header_1", mod_00_header_server)
+  moduleServer("01_intro_1", mod_01_intro_server)
   moduleServer("markdown_example_1", mod_markdown_example_server)
```

Comment:
Can remove this file as was just an example of a markdown module (which the intro now is)